packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -628,6 +628,7 @@ const ScrollAreaViewportImpl = forwardRefWithAs<typeof VIEWPORT_DEFAULT_TAG>(
             // get around this with an extra wrapper with `display: table`.
             // https://blog.alexandergottlieb.com/overflow-scroll-and-the-right-padding-problem-a-css-only-solution-6d442915b3f4
             display: 'table',
+            width: '100%',
```

Comment:
Is there any other solution for this? The `display: table` does not work with `min-height` in Firefox.. Example usage: Page that uses ScrollArea and has a centered login form. To be able to center form, I have to set `min-width: 100%` on viewport, and it works fine in chrome, but not in firefox.

Reply:
@vdjurdjevic, we are currently in the process of giving `ScrollArea` some attention so it's possible this may change. Regardless we'll keep an eye on it.

cc @jjenzz

Reply:
Great :) Take a look at https://github.com/sakhnyuk/rc-scrollbars
This works like a charm :)